frameworks/CFML/cfml-lucee/cfml-lucee.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ortussolutions/commandbox:latest
+
+COPY ./src/server.json /app/server.json
+COPY ./src/.cfconfig.json /app/.cfconfig.json
+
+RUN box install commandbox-cfconfig --verbose --force --debug
+
+RUN ${BUILD_DIR}/util/warmup-server.sh
+
+RUN export FINALIZE_STARTUP=true;$BUILD_DIR/run.sh;unset FINALIZE_STARTUP
+
```

Comment:
Though it's not needed yet, can you `EXPOSE 8080` here. When we move to the new toolset, this will be necessary.

Reply:
Exposed